tb_pulumi/__init__.py

```diff
@@ -149,6 +154,7 @@ def finish(self, outputs: dict[str, Any], resources: dict[str, pulumi.Resource |
 
         # Register outputs both with the ThunderbirdPulumiProject and Pulumi itself
         self.resources = resources
+        self.project.resources[self.name] = self.resources
```

Comment:
This solves a bug where the component resource knows what it contains, but the project around it does not.